var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -35,17 +35,23 @@ class Extrae(Package):
        OpenMP, CUDA, OpenCL, pthread, OmpSs"""
     homepage = "https://tools.bsc.es/extrae"
     url      = "https://ftp.tools.bsc.es/extrae/extrae-3.4.1-src.tar.bz2"
+    version('3.7.1', sha256='95810b057f95e91bfc89813eb8bd320dfe40614fc8e98c63d95c5101c56dd213', preferred=True)
```

Comment:
No need for `preferred=True` here. Spack always defaults to the latest stable release.